mender-deb-package

```diff
@@ -61,21 +61,28 @@ checkout_repo() {
 
 install_go() {
   local GOLANG_VERSION=1.21.0
+  local GOLANG_ARCH=amd64
   local golang_version_set
+  if [ $ARCH = "armhf" -a $OS_DISTRO = "debian" ]; then
+    local GOLANG_ARCH=armv6l
+  fi
 
   golang_version_set=$(sed -n '/[^ ]*GOLANG_VERSION:[ "0-9]/{s/.*GOLANG_VERSION:[ ]*\(["0-9\.]*\).*/\1/p;q}' .gitlab-ci.yml)
   golang_version_set=${golang_version_set//\"/}
   GOLANG_VERSION=${golang_version_set:-${GOLANG_VERSION}}
 
-  wget -q https://dl.google.com/go/go$GOLANG_VERSION.linux-amd64.tar.gz \
-     && tar -C /usr/local -xzf go$GOLANG_VERSION.linux-amd64.tar.gz
+  wget -q https://dl.google.com/go/go$GOLANG_VERSION.linux-${GOLANG_ARCH}.tar.gz \
+     && tar -C /usr/local -xzf go$GOLANG_VERSION.linux-${GOLANG_ARCH}.tar.gz
   export GOPATH="/root/go"
   export PATH="$PATH:/usr/local/go/bin"
-  rm -vf "go$GOLANG_VERSION.linux-amd64.tar.gz"
+  rm -vf "go$GOLANG_VERSION.linux-${GOLANG_ARCH}.tar.gz"
 }
 
 get_os_version() {
   OS_DISTRO="$(. /etc/os-release && echo "$ID")"
+  if [ $OS_DISTRO = "raspbian" ]; then
+      OS_DISTRO="debian"
+  fi
```

Comment:
This is fine but it deserves a comment for the future you or the future me :nerd_face: